src/gui/AboutDialog.cpp

```diff
@@ -54,6 +54,9 @@ AboutDialog::AboutDialog(QWidget* parent)
 
     QString debugInfo = "KeePassXC - ";
     debugInfo.append(tr("Version %1\n").arg(KEEPASSX_VERSION));
+#ifdef KEEPASSXC_NIGHTLY_BUILD
+    debugInfo.append(tr("Build Type: Nightly\n"));
```

Comment:
I would just add this in brackets after the version string. Also maybe use "Snapshot" / "SNAPSHOT" instead of "Nightly"??)

Reply:
Corrected in last push